packages/nimble-components/src/wafer-map/index.ts

```diff
@@ -246,12 +268,63 @@ export class WaferMap extends FoundationElement {
             } else if (this.waferMapUpdateTracker.requiresDrawnWaferUpdate) {
                 this.renderer.drawWafer();
             }
+            await this.drawWorkerWafer();
             this.zoomHandler.connect();
         } else if (this.waferMapUpdateTracker.requiresRenderHoverUpdate) {
             this.renderer.renderHover();
         }
     }
 
+    private async drawWorkerWafer(): Promise<void> {
```

Comment:
I think I would just call this `drawWafer`...the fact that a worker is being used is an implementation detail.

Reply:
indeed, I renamed it